app/api/event_invoices.py

```diff
@@ -0,0 +1,105 @@
+from flask_rest_jsonapi import ResourceDetail, ResourceList, ResourceRelationship
+from marshmallow_jsonapi.flask import Schema, Relationship
+from marshmallow_jsonapi import fields
+
+from app.api.helpers.utilities import dasherize
+from app.api.helpers.permissions import is_admin
+from app.models import db
+from app.models.event import Event
+from app.models.user import User
+from app.models.event_invoice import EventInvoice
+
+
+class EventInvoiceSchema(Schema):
+
+    class Meta:
+        type_ = 'event_invoice'
+        self_view = 'v1.event_invoice_detail'
+        self_view_kwargs = {'id': '<id>'}
+        self_view_many = 'v1.event_invoice_list'
```

Comment:
@magdalenesuo agreed. Removing /event-invoices will require you to remove this since this endpoint no longer exist.